coral/README.md

```diff
@@ -24,10 +24,22 @@
 
 ℹ️ Coral uses `pnpm run` as a package manager. Read their official documentation [how to install](https://pnpm.io/installation) it.
```

Comment:
```suggestion
ℹ️ Coral uses `pnpm run` as a package manager. Read their official documentation [how to install](https://pnpm.io/installation) pnpm.
```

Reply:
Any specific versions of node/pnpm are required to run klaw

Reply:
We're running the lasted PNPM, that is be installed automatically if users install it new. But I can make a note about what major version we're using at least. 

The node version is set per `.npmrc`, so users would get notified if they need a different node version. I will add that in the docs and link the npmrc file, then it's easy to keep the versions in sync!